Planner/src/Planner.java

```diff
@@ -125,28 +125,58 @@ public void actionPerformed(ActionEvent arg0) {
 			yearList.addElement(i);
 		}
 		
+		DefaultComboBoxModel<String> hourList = new DefaultComboBoxModel<>();
+		JList listWithHours = new JList(hourList);
+		for (int i = 0; i <= 24; i++) {
+			hourList.addElement(String.format("%02d", i));
+		}
+		
+		DefaultComboBoxModel<String> minuteList = new DefaultComboBoxModel<>();
+		JList listWithMinutes = new JList(minuteList);
+		for (int i = 0; i <= 60 ; i++) {
+			minuteList.addElement(String.format("%02d", i));
+		}
+		
 		// click on button Add new task
 		addTaskButton.addActionListener(new ActionListener() {
 			@Override
 			public void actionPerformed(ActionEvent arg0) {
 				// TODO Auto-generated method stub
 				JComboBox selectTask = new JComboBox(tasksList);
 				JComboBox assignedTo = new JComboBox(usersList);
-			    JComboBox days = new JComboBox(dayList);
-			    JComboBox months = new JComboBox(monthList);
-			    JComboBox years = new JComboBox(yearList);
+			    JComboBox dayStart = new JComboBox(dayList);
+			    JComboBox monthStart = new JComboBox(monthList);
+			    JComboBox yearStart = new JComboBox(yearList);
+			    JComboBox hourStart = new JComboBox(hourList);
+			    JComboBox minuteStart = new JComboBox(minuteList);
+			    JComboBox dayEnd = new JComboBox(dayList);
+			    JComboBox monthEnd = new JComboBox(monthList);
+			    JComboBox yearEnd = new JComboBox(yearList);
+			    JComboBox hourEnd = new JComboBox(hourList);
+			    JComboBox minuteEnd = new JComboBox(minuteList);
 				JTextField start = new JTextField(30);
 			    JTextField end = new JTextField(30);
 				JTextField taskName = new JTextField(25);
 			    JTextField taskDescription = new JTextField(30);
 			    JCheckBox repeatableEachYear = new JCheckBox("Repeatable each year");
 			    JPanel taskPanel = new JPanel();
-			    JPanel datePanel = new JPanel();
+			    JPanel datePanelStart = new JPanel();
+			    JPanel datePanelEnd = new JPanel();
 			    
-			    start.setText(years.getSelectedItem().toString() + "-" + months.getSelectedItem().toString() + "-" + days.getSelectedItem().toString());
-			    datePanel.add(days);
-			    datePanel.add(months);
-			    datePanel.add(years);
+			    start.setText(yearStart.getSelectedItem().toString() + "-" + monthStart.getSelectedItem().toString() + "-" + 
+			    dayStart.getSelectedItem().toString() + " " + hourStart.getSelectedItem().toString() + ":" + minuteStart.getSelectedItem().toString() );		    
```

Comment:
You can add at the end + "00" for the seconds.
dayStart.getSelectedItem().toString() + " " + hourStart.getSelectedItem().toString() + ":" + minuteStart.getSelectedItem().toString() + "00");
Otherwise, I think there will be an error on the sql part and maybe the date won't be well saved in mysql ?
 Add this + "00" for the end date as well.